publishing.gradle

```diff
@@ -0,0 +1,29 @@
+// Sets up publishing of release artifacts.
+// Note: Keep this script in sync across all airbnb open source projects.
+apply plugin: 'com.vanniktech.maven.publish'
+
+
+// https://vanniktech.github.io/gradle-maven-publish-plugin/other/
+publishing {
```

Comment:
Thank you for this file, I ended up using this for Showkase publishing as well 🙏